doc/howto/cluster_groups.md

```diff
@@ -27,6 +27,13 @@ To assign `server1` to the `gpu` group and also keep it in the `default` group,
 
     lxc cluster group assign server1 default,gpu
 
+To add a cluster member to a specific group, use `lxc cluster group add` command.
+This command adds a cluster member to the specified group.
+
+For example, to add `cluster:node1` to the `newgroup` group , use the following command:
```

Comment:
Please can you use `server1` rather than `cluster:node1` to be consistent and also to avoid using "node" terminology we dont use in user face docs/messages. Thanks